Sources/Extensions/UIKit/UIDeviceExtensions.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIDeviceExtensions.swift
+//  SwifterSwift
+//
+//  Created by Anton Novoselov on 31/12/2017.
+//  Copyright © 2017 SwifterSwift
+//
+
+#if os(iOS) || os(tvOS)
+import UIKit
+public extension UIDevice {
+    /// Show country code of current locale for device
```

Comment:
Need to prefix this with `SwifterSwift`. Please read [Adding new extensions, Adding documentation](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md#adding-new-extensions). Use existing extensions as a reference.